### ISSUE TITLE
Incompatibility between re-frisk 1.5.0 and re-frame-10x 1.0.2

### ISSUE THREAD

**Issue by MarkN3DTI** [flexsurfer/re-frisk#63]
Version 1.5.0 of re-frisk and version 1.0.2 of re-frame-10x use incompatible versions of `day8.reagent.impl.component.cljs`, which produces the following compile failure when I try to run `lein cljsbuild once`:

```
clojure.lang.ExceptionInfo: Invalid :refer, var day8.reagent.impl.component/patch-custom-wrapper does not exist in file resources/public/js/out/day8/re_frame_10x.cljs {:tag :cljs/analysis-error}
```

**Comment by superstructor** [comment]
@flexsurfer It is all good to be re-using the tracing monkey patches from re-frame-10x but please rename the namespace so that there is no possibility of collisions; e.g. `re-frisk.inlined-deps.day8.reagent.impl....` or whatever style you prefer as long as it is not in namespaces expected to be controlled by Day8's re-frame project's such as `day8.*`, `re-frame.*` etc.

Also there is day8/re-frame-10x#115 for designing a proper API in reagent to do this, rather than overriding internal implementations of reagent functions but unfortunately we don't have time to look at that at the moment.

**Comment by flexsurfer** [comment]
thanks, didn't expect it's possible to use both tools at the same time :) i'll fix that soon

**Comment by flexsurfer** [comment]
fixed  1.5.1
